src/events/EventsApp.js

```diff
@@ -90,17 +84,23 @@ const EventsApp = observer(class App extends React.Component {
         this.view.ui.add(searchExpand, "top-right");
         this.view.ui.add(legend, "bottom-right");
         this.view.ui.move("zoom", "top-right");
-        App._fixLayerOrder(this.view);
+        this._fixLayerOrder(this.view);
       });
   }
 
-  static _fixLayerOrder(mapView) {
+  _fixLayerOrder(mapView) {
     const layers = mapView.map.layers;
     const iSpeed = layers.items.findIndex(lyr => lyr.id === 'speed');
     if (iSpeed > 0) {
       const speedLayer = layers.getItemAt(iSpeed);
       layers.reorder(speedLayer, 0);
     }
+    if (eventsConfig.streetNamesBelowIcons) {
+      this.store.doAfterLayersLoaded = () => {
+        const streetNamesLayer = mapView.map.basemap.referenceLayers.pop();
+        layers.add(streetNamesLayer, 1);
+      }
+    }
```

Comment:
It would be nice to have a better way to do this, but for now I can't find any alternatives...
Or we can use some transparency for street names layer.. It's simple and safe.